Resources/Prototypes/_Impstation/Entities/Mobs/Customization/Markings/moth.yml

```
@@ -0,0 +1,457 @@
+
+- type: marking
+  id: ChimeraArmLeftMoth
+  bodyPart: LArm
+  markingCategory: LeftArm
+  speciesRestriction: [Moth]
+  followSkinColor: false
+  coloring:
+    default:
+      type:
+        !type:SkinColoring
+          negative: true
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 10

(indentation)